### ISSUE TITLE
Tab "Place history" uses only current GOV hierarchy

### ISSUE THREAD

**Issue by hartenthaler** [vesta-webtrees-2-custom-modules/vesta_shared_places#120]
The "GOV Hierarchy" entries in the "Place history" are great! Thank you! But they are showing always the GOV hierarchy of today. This is independent of the configuration in control panel.

I would prefer to see the hierarchy at the time a "GOV Hierarchy" entry belongs to.
![Screenshot (127)](https://user-images.githubusercontent.com/6324175/166330407-4c2915e5-f70c-46ae-9c44-fa8984dbc1eb.png)

**Comment by ric2016** [comment]
I think this is the same issue as [here](https://github.com/vesta-webtrees-2-custom-modules/vesta_gov4webtrees/issues/79):
The GOV hierarchy is supposed to show up in these entries 'as usual' (see my screenshot in the linked issue).

There seems to be a problem with that, likely in the respective ajax requests.

**Comment by hartenthaler** [comment]
The GOV hierarchy is shown as expected for 'regular' facts (on the facts & events tab). At the moment I use the option to show both the historic and the actual hierarchy.

**Comment by ric2016** [comment]
Ok, please check whether there are failing ajax requests on the place history page.

**Comment by ro-la** [comment]
I tried to reset all my Vesta configuration options - deleted all Vesta moduls, after this in display Modules I deleted the preferences for all Vesta modules. 

After installing the modules ther was no change.

Other facts&events are dispalyed correctly. 

@hartenthaler I am glad I am not the only person with this problem :-)

**Comment by ric2016** [comment]
Yes, I'll have to investigate this further - Ajax requests finish sucessfully, but contain empty html where the GOV hierarchy is supposed to be.

**Comment by ric2016** [comment]
This occurs for GOV-Ids defined on Shared Places, if the module setting "Additionally link shared places via name" is unchecked.

Will be fixed soon, workaround in the meantime is to check that setting.
